util/wordpress.js

```diff
@@ -247,7 +247,7 @@ function installTheme(generator, config, done) {
 
 };
 
-function setupTheme(generator, config, done) {
+function setupTheme(generator, config, cb) {
```

Comment:
will do. i couldn't quite get it to work the same using `async()` in this way. hence the change. i may do a little more experimenting and see if we can avoid the change.